build/dependencies.props

```diff
@@ -332,56 +331,56 @@
-    <ExternalDependency Include="System.Composition.AttributedModel" Version="1.1.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Composition.Convention" Version="1.1.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Composition.Hosting" Version="1.1.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Composition.Runtime" Version="1.1.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Composition.TypedParts" Version="1.1.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Composition" Version="1.1.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Configuration.ConfigurationManager" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.Buffers" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.CodeDom" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Collections.Immutable" Version="1.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.ComponentModel.Annotations" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Composition.AttributedModel" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Composition.Convention" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Composition.Hosting" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Composition.Runtime" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Composition.TypedParts" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Composition" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Configuration.ConfigurationManager" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
     <ExternalDependency Include="System.Data.SqlClient" Version="4.3.1" Source="$(DefaultNuGetFeed)" Private="true">
       <!-- This version is used by EF's 1.1 benchmark project -->
       <NoWarn>KRB2004</NoWarn>
       <VariableName>Benchmarks_11_SystemDataSqlClientPackageVersion</VariableName>
     </ExternalDependency>
-    <ExternalDependency Include="System.Data.SqlClient" Version="4.4.0" Source="$(DefaultNuGetFeed)">
+    <ExternalDependency Include="System.Data.SqlClient" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true">
```

Comment:
Nooo. 😞

Reply:
@smitpatel is there a reason we shouldn't upgrade?

Reply:
No reason. I got confused by diff and thought that we are upgrading the version which is being used in EF1.1 benchmarks.
All good.